include/rdma/fi_log.h

```diff
@@ -87,7 +85,7 @@ void fi_log(const struct fi_provider *prov, enum fi_log_level level,
 #define FI_INFO(prov, subsystem, ...)					\
 	FI_LOG(prov, FI_LOG_INFO, subsystem, __VA_ARGS__)
 
-#if ENABLE_DEBUG
+#if defined ENABLE_DEBUG && ENABLE_DEBUG > 0
```

Comment:
To move forward with this patch, please revert this change.  If you want, please submit the changes to fi_log.h as a separate commit